source/faq.txt

```diff
@@ -0,0 +1,133 @@
+===
+FAQ
+===
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+How Is the Kotlin Driver Different from Kmongo?
+-----------------------------------------------
+
+The Kotlin driver is the official MongoDB driver for Kotlin. It is
+developed by the MongoDB team and provides a native API for Kotlin
+applications to connect to MongoDB and work with data. It is implemented
+by wrapping the :driver:`MongoDB Java driver </java-drivers/>`.
+
+`KMongo <https://litote.org/kmongo/>`__ is a popular community-developed library
+for working with MongoDB from Kotlin applications.
+It is also a wrapper around the Java driver.
+It was created prior to the creation of the official Kotlin driver to serve
+the needs of the Kotlin community.
+
+The Kotlin driver was developed in collaboration with the creator of KMongo,
```

Comment:
Maybe something to explain why we built this when Kmongo already existed -- like "The MongoDB server-side Kotlin driver was developed to give users an officially-supported driver" or something like that?